packages/circle-demo-webapp/app/components/TokenSelect/TokenSelectItem.tsx

```diff
@@ -10,18 +10,15 @@ export interface TokenSelectItemProps {
 /** A token balance for an on-chain account */
 export function TokenSelectItem({ balance }: TokenSelectItemProps) {
   return (
-    <div className="flex items-center space-x-4">
+    <div className="flex items-center space-x-2 pr-4">
       <TokenIcon
         symbol={balance.token.symbol.split('-')[0]}
-        size={40}
+        size={24}
         variant="branded"
         className="flex-shrink-0"
       />
-
-      <div>
-        <p className="text-base font-medium text-gray-900">
-          {balance.amount} {balance.token.symbol}
-        </p>
+      <div className="text-sm text-muted-foreground">
+        {balance.amount} {balance.token.symbol}
```

Comment:
@avkos for dark theme support we can't use class names that define color, with ie. `foreground` the theme config will decide on the colors

Reply:
made changes in my branch